start-firecracker.sh

```diff
@@ -6,8 +6,8 @@ RO_DRIVE="$PWD/xenial.rootfs.ext4"
 KERNEL="$PWD/vmlinux"
 TAP_DEV="fc-${SB_ID}-tap0"
 
-KERNEL_BOOT_ARGS="panic=1 pci=off reboot=k tsc=reliable quiet 8250.nr_uarts=0 ipv6.disable=1 $R_INIT"
-#KERNEL_BOOT_ARGS="console=ttyS0 reboot=k panic=1 pci=off nomodules ipv6.disable=1 $R_INIT"
+KERNEL_BOOT_ARGS="panic=1 pci=off reboot=k tsc=reliable quiet 8250.nr_uarts=0 ipv6.disable=1"
```

Comment:
Why is R_INIT deleted? Is this change related to updated to v0.18.0? If yes, can you please state that in your commit message?

Reply:
`Updated unused env variable from start_firecracker.sh.` from PR description was meant to describe this change.
I will update the description to `Removed empty env variable (R_INIT) from start_firecracker.sh.`. I will add this in the commit message also.